internal/proto/controller/api/services/v1/session_service.proto

```diff
@@ -63,6 +63,9 @@ message ListSessionsRequest {
   string scope_id = 1; // @gotags: `class:"public"`
   bool recursive = 20 [json_name = "recursive"]; // @gotags: `class:"public"`
   string filter = 30 [json_name = "filter"]; // @gotags: `class:"public"`
+  // Experimental. By default only non-terminated (i.e. pending, active, canceling) are returned.
```

Comment:
The "Experimental" keyword strikes me as odd here, considering it's the default behaviour right now. 

As I understand it, we're eventually deprecating this behaviour, so wouldn't this be a legacy/compatibility option or something like that?

Reply:
Experimental is on the flag though, right? The current behavior is changing, so rather than just change it, there is a flag that may or may not continue existing, hence experimental.

I also don't think it's legacy/compatibility, especially given that we may never deprecate it and may simple officially support it once we are cleaning out terminated sessions with regularity.